### PR TITLE
Week1-AmirHossein-Exercises 

### PR DESCRIPTION
**Pull request by amirhshad**

### CODE REVIEW

File: Week1/homework/js-exercises/dogGallery.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    const xhr = new XMLHttpRequest();
+    const li = document.createElement('li');
+    const image = document.createElement('img');
+    const ul = document.getElementById('list');
+    const url = 'https://dog.ceo/api/breeds/image/random'
+    xhr.open('GET', url);
+    xhr.send();
+    xhr.onreadystatechange = processRequest;
+
+    ul.appendChild(li.appendChild(image))
```
Comment (n4cr) [inline Week1/homework/js-exercises/dogGallery.js:14]: Does this work? I guess it should be 
```
li.appendChild(image)
ul.appendChild(li)
```
as two separate lines.

File: Week1/homework/js-exercises/dogGallery.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+
+function axiosMethod() {
+    const li = document.createElement('li');
+    const image = document.createElement('img');
+    const ul = document.getElementById('list');
+    ul.appendChild(li.appendChild(image))
+    axios
+    .get('https://dog.ceo/api/breeds/image/random')
+    .then(function(response) {
```
Comment (n4cr) [inline Week1/homework/js-exercises/dogGallery.js:45]: here you're using a promise. as you can see the `then` and `catch` are the functions to handle the result asynchronously

**Review by n4cr — APPROVED** [review]
Looks pretty good. Please look at my two comments

File: Week1/homework/js-exercises/dogGallery.js (hunk 1 above)
Comment (amirhshad) [reply Week1/homework/js-exercises/dogGallery.js:14]: Yeah, it's working. Also it's possible to write it in two lines

**Pull request merged**